src/scripts/main.js

```diff
@@ -5,4 +5,24 @@ const people = require('./lib/people');
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('tbody');
+
+for (const person of people) {
+  table.insertAdjacentHTML('beforeend', `
+  <tr>
+    <td>${person.name}</td>
+    <td>${getGender(person)}</td>
+    <td>${person.born}</td>
+    <td>${person.died}</td>
+    <td>${person.died - person.born}</td>
+    <td>${Math.ceil(person.died / 100)}</td>
```

Comment:
use destructuring

Reply:
> use destructuring

![image](https://user-images.githubusercontent.com/109792077/189519756-bab29f87-04e6-47be-88c4-ea94e7c939a6.png)
In this way, I have parsing erron. But I will try in different one.